build.gradle

```diff
@@ -0,0 +1,95 @@
+buildscript {
+    repositories {
+        maven {
+            name = 'forge'
+            url = 'http://files.minecraftforge.net/maven'
+        }
+        maven {
+            name = 'sponge'
+            url = 'http://repo.spongepowered.org/maven'
+        }
+    }
+    dependencies {
+        classpath 'net.minecraftforge.gradle:ForgeGradle:2.2-SNAPSHOT'
+        classpath 'org.spongepowered:mixingradle:0.3-SNAPSHOT'
+    }
+}
+
+plugins {
+    id 'java'
+
+    id 'idea'
+    id 'eclipse'
```

Comment:
FG automatically applies all of them (applying `java`, `idea`, `eclipse` is redundant)